### ISSUE TITLE
'--new-from-rev=master' flag not taking effect

### ISSUE THREAD

**Issue by jlucktay** [golangci/golangci-lint-action#30]
We run this in some of our projects, and thank you all for the tool; it's fantastic!

I am noticing that the `issues.new-from-rev: master` setting in our `.golangci.yml` configuration file isn't effective, and this warning is in our GH Actions log:

```
level=warning msg="[runner] Can't process result by diff processor: can't prepare diff by revgrep: could not read git repo: error executing git diff \"master\" \"\": exit status 128"
```

Does it seem like the Action can't find the current revision hash to diff against `master`?

**Comment by CyrusJavan** [comment]
Use `only-new-issues: true` in your action yml instead. I had your exact issue, using `only-new-issues: true` will accomplish the same effect as `issues.new-from-rev: master`
```
      - name: golangci-lint
        uses: golangci/golangci-lint-action@v1
        with:
          version: v1.26
          only-new-issues: true
```

**Comment by jlucktay** [comment]
OK, thanks @CyrusJavan!
